scripts/templates/valddi.cpp.mako

```diff
@@ -60,6 +60,16 @@ namespace ur_validation_layer
 
             %endfor
             %endfor
+            %if func_name in th.get_event_wait_list_functions(specs, n, tags):
+            if (phEventWaitList != NULL && numEventsInWaitList > 0) {
+                for (uint32_t i = 0; i < numEventsInWaitList; ++i) {
+                    if (*(phEventWaitList + i) == NULL) {
```

Comment:
would `phEventWaitList[i]` work here instead?

Reply:
Changed it in the latest commit :+1: